### DISCUSSION TITLE
admin bar missing translations

### DISCUSSION THREAD

**Discussion by cbratschi** [payloadcms/payload#15254] [Feature Requests & Ideas]
A couple of strings are hardcoded and therefore cannot be translated:

- 'Edit ' is hardcoded (see https://github.com/payloadcms/payload-admin-bar/blob/main/src/index.tsx#L278)
- 'New ' is hardcoded (see https://github.com/payloadcms/payload-admin-bar/blob/main/src/index.tsx#L313)
- 'Logout' is hardcoded (see https://github.com/payloadcms/payload-admin-bar/blob/main/src/index.tsx#L372)

There should be some properties to pass custom values.

**Comment by JessRynkar** [comment]
Hi @cbratschi, I agree this is definitely needed. I'm going to convert this to a feature request, it's a small one that someone from the community may want to work on. Thanks for opening this.

**Reply by cbratschi** [reply]
We implemented our own custom frontend component quite some time ago which is fully translated. But still needed by other users.
